src/OpenTK/Platform/NativeWindowBase.cs

```diff
@@ -347,6 +348,18 @@ protected void UpdateModifierFlags(KeyModifiers mods)
                     OnKeyUp(Key.ShiftRight);
                 }
             }
+
+            if (command)
+            {
+                OnKeyDown(Key.Command, KeyboardState[Key.AltLeft]);
```

Comment:
Is the use of the alt left key here intentional?

Reply:
It's intentional, copying the pattern from those above.

Modifiers on OS-X / Cocoa don't generate a keydown state event per-se, but rather are passed as flags in a window event message (Including ALT).
If I understand the intent of the original code correctly, the virtual LeftAlt key is being used to define that any given modifier was valid at the last keypress event, and the modifiers flags are used to determine which.

Reply:
That.... doesn't seem to make much sense to me. Why would a modifier not be valid?

Reply:
Because a modifier isn't a key & generates no events.

We can therefore hold one key and a modifier. Release the modifier while holding the key & press another.
The first is now invalid, but no keypress events have been generated.

If this don't make sense it's because its from the phone....

Reply:
Oh, that flag just sets the KeyDownArgs.IsRepeat field to determine if its a key repeat event when holding down the key for a longer time. So yes, should be corrected.